clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -77,56 +72,45 @@ func init() {
 	}
 }
 
-type apiCall struct {
+type apiCallMetric struct {
```

Comment:
Can we move it down? Usually you define a type and immediately below the methods for that type. Can we stick to such convention?

Reply:
Or ideally, given this file has 300 lines of code, can we extract the apiCallMetric type to a separate file?

Reply:
I plan to send a separate PR that moves each SLO measurement to its own package (for the reason stated in PR's description). Once that's done I'd split this into two files. WDYT?